src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -231,14 +238,34 @@ public void showOutputSummary() {
         isFailure ? "\u001B[1;31m" : "\u001B[1;32m",
         isFailure ? "\u001B[1;35m" : "\u001B[1;32m", log);
 
-    uploadOutputSummary(failed, completedChecks);
+    // Comment out here, for this step is used directly in the report Task
+    // uploadOutputSummary(failed, completedChecks);
 
     if (isFailure && GatorGradleConfig.get().shouldBreakBuild()) {
       throw new GradleException(
           StringUtil.color(StringUtil.BAD, "Grading checks failed -- scroll up for failures"));
     }
   }
 
+  /**
+   * Return the completed checks.
+   */
+  public List<CheckResult> getCompletedchecks() {
+    return completedChecks;
+  }
+
+
+  /**
+   * Return the failed checks.
+   */
+  public List<CheckResult> getFailed() {
```

Comment:
I'm torn about this method -- it seems like it'd be fairly useful, but also just a shortcut and dependent on an internal encapsulated member variable. I think it is probably best to remove this method as well, and rely on doing this check where you need it, especially because in some cases you may not be working with `completedChecks` but some other list.